tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/readonly/TupleVsArrayReadonlyErrorConsistencyTest.java

```diff
@@ -0,0 +1,47 @@
+package org.ballerinalang.test.types.readonly;
```

Comment:
Missing the header

Reply:
Fixed in 6e9b1d9980383e38660e4e1193ee3a3b647a8928. It's an obvious mistake in my part. Will make sure to double check in the future.